sdk/formrecognizer/azure-ai-formrecognizer/swagger/README.md

```diff
@@ -64,16 +65,26 @@ namespace: azure.ai.formrecognizer.v2_1
 output-folder: $(python-sdks-folder)/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_generated/v2_1
 ```
 
-## Release 3.1-preview
+## Release 3.1
 
 These settings apply only when `--tag=release_2022_08_31` is specified on the command line.
 
 ``` yaml $(tag) == 'release_2022_08_31'
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs-pr/b1e128f86879ba5a07502abe60577300eb9ec392/specification/cognitiveservices/data-plane/FormRecognizer/stable/2022-08-31/FormRecognizer.json?token=GHSAT0AAAAAABUYDSJIPTM5ED3IVOUKSYP2YX24EKA
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/cognitiveservices/data-plane/FormRecognizer/stable/2022-08-31/FormRecognizer.json
```

Comment:
nit: we should use commits for referencing the swagger instead of pointing to main